PopPUNK/refine.py

```diff
@@ -204,8 +242,35 @@ def refineFit(distMat, sample_names, start_s, mean0, mean1,
 
     return start_point, optimal_x, optimal_y, min_move, max_move
 
+def expand_cugraph_network(G, G_extra_df):
+    """Reconstruct a cugraph network with additional edges.
+    
+    Args:
+        G (cugraph network)
+            Original cugraph network
+        extra_edges (cudf dataframe)
+            Data frame of edges to add
 
-def growNetwork(sample_names, i_vec, j_vec, idx_vec, s_range, score_idx, thread_idx = 0, use_gpu = False):
+    Returns:
+        G (cugraph network)
+            Expanded cugraph network
+    """
+    # load CUDA libraries
+    if not gpu_lib:
+        sys.stderr.write('Unable to load GPU libraries; exiting\n')
+        sys.exit(1)
+    G_vertex_count = G.number_of_vertices()-1
+    G_original_df = G.view_edge_list()
+    if 'src' in G_original_df.columns:
+        G_original_df.columns = ['source','destination']
+    G_df = G_original_df.append(G_extra_df)
+    del G_original_df
+    del G_extra_df
```

Comment:
Would prefer not to delete an arg within the function

Reply:
Removed in 8fc9763.